tests/cli/deployment/test_deployment_build.py

```diff
@@ -480,19 +483,70 @@ def test_poorly_formed_entrypoint_raises_correct_error(
             expected_output_contains=f"Your flow entrypoint must include the name of the function that is the entrypoint to your flow.\nTry {entrypoint}:<flow_name>",
         )
 
-    def test_entrypoint_that_does_not_point_to_flow_raises_error(
-        self, monkeypatch, tmp_path
-    ):
+    def test_entrypoint_that_does_not_point_to_flow_raises_error(self, tmp_path):
+        code = """
         def fn():
             pass
+        """
+        fpath = tmp_path / "dog.py"
+        with open(fpath, "w") as f:
+            f.write(dedent(code))
+
+        name = "TEST"
+        entrypoint = f"{fpath}:fn"
+        cmd = [
+            "deployment",
+            "build",
+            "-n",
+            name,
```

Comment:
I'm not sure it's worth forcing this onto separate lines with the trailing comma
```suggestion
            name
```